master/internal/api_user.go

```diff
@@ -30,7 +30,9 @@ const determinedName = "determined"
 var (
 	errExternalSessions = status.Error(codes.PermissionDenied, "not enabled with external sessions")
 	errUserNotFound     = status.Error(codes.NotFound, "user not found")
-	latinText           = regexp.MustCompile("[^[:graph:]\\s]")
+	errUserNotFoundPerm = status.Error(codes.NotFound,
```

Comment:
nit: could this be based off of a util to add this messaging to existing error msgs? you should be able to use the same in other endpoints

Reply:
some of our tests might be checking for these values, now or in the future, and having a consistent way of generating them helps those as well

Reply:
Resolved by adding the `MakePermErrors` function in apiutils, and then generating all its references in the source code + tests as `MakePermErrors(err)`